src/client/containers/LandingPage/LandingPage.Container.js

```diff
@@ -1,11 +1,46 @@
 import React from 'react';
 import './LandingPage.Style.css';
+import HeroImage from '../../components/HeroImage/HeroImage.component';
+import Carousel from '../../components/Carousel/Carousel.component';
+import ProductView from '../../components/ProductView/ProductView.component';
 
 const LandingPageContainer = () => {
+  const [products, setProducts] = React.useState();
+  const [productsImages, setProductsImages] = React.useState();
+  const [categories, setCategories] = React.useState();
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  React.useEffect(() => {
+    fetch('/api/products')
+      .then((res) => res.json())
+      .then((data) => {
+        setProducts(data);
+        setProductsImages(data.map((product) => product.picture));
+        setIsLoading(true);
+      });
+    fetch('/api/categories')
+      .then((res) => res.json())
+      .then((data) => setCategories(data));
+  }, []);
+
   return (
-    <div>
-      <h1>Landing page Container</h1>
-    </div>
+    <main>
+      <div className="hero-image">
+        {isLoading && <HeroImage heroText="WELCOME" />}
+      </div>
+      <div>
+        {isLoading && <Carousel imageArray={productsImages} show={3} />}
+      </div>
+      <div>
+        {isLoading && (
+          <ProductView
+            products={products}
+            productsPerPage={6}
```

Comment:
I think we need change size from css. Because every row has 3 item in productview. If I make it 12 then we need to add more products. Because there is no item left in 2.page or other pages.